src/controller/java/src/chip/devicecontroller/model/NodeState.java

```diff
@@ -44,8 +43,7 @@ private void setDataVersion(int endpointId, long clusterId, long dataVersion) {
     }
   }
 
-  // Called from native code only, which ignores access modifiers.
-  private void addAttribute(
+  public void addAttribute(
```

Comment:
please set this as protected, so that ReportCallbackJni.Java can still can it from same package, and other package cannot see this.

Reply:
To solve this problem, we modified the NodeState method to be called directly from jni.